lib/i18n_fixes.rb

```diff
@@ -67,28 +67,3 @@ def locale
     end
   end
 end
-
```

Comment:
This information should be in the commit message.

What prompted the change? Failing specs?